docs/3.0.0-beta.x/guides/schedule-article.md

```diff
@@ -0,0 +1,80 @@
+# Schedule article
+
+This guide will explain how to create an article schedule system.
+
+## Introduction
+
+This guide will use this [draft system](.draft.md) guide. So first read it to understand the draft/published status and the Article API that we will use in this guide.
+
+What we want here is to be able to set a publication date for an article, and at this date, switch the draft status to published.
+
+## Example
+
+For this example, we will have to add a `published_at` attribute to the Article Content Type.
+
+- Click on the Content Type Builder link in the left menu
+- Select the Article Content Type
+- Add another field
+  - `date` attribute named `published_at` with `datetime` type
+
+And add some data with different dates and status to be able to see a plublication happen.
+Some entries need to have `draft` status and a `published_at` date lower that your current date.
+
+The goal will be to verify every minute if there is `draft` articles that have a `published_at` lower that the current date.
+
+## Create a CRON task
+
+To execute a function every minutes, we will use CRON tasks.
```

Comment:
`To execute a function every minute, we will use a CRON task.`

Side note, may want to include a `:::: note` here stating this will not work at scale using the built in Strapi cron solution.

Reply:
What is the exact message you would like to see?

Reply:
🤔 Something along the lines of `Please note that Strapi's built in cron feature will not work if you plan to use pm2 or node based clustering. You will need to execute these cron tasks outside of Strapi`

Reply:
The reasoning is because if I duplicate the same cron code multiple times it will fire this same task at the same time and effectively introduce a race condition. @alexandrebodin and I were discussing this yesterday I think in slack.

Reply:
Perfect!